docs/MACOS_INSTALL.md

```diff
@@ -0,0 +1,214 @@
+shasum -a 256 -c RamaLama-VERSION-macOS-Installer.pkg.sha256
+
+# Install
+sudo installer -pkg RamaLama-VERSION-macOS-Installer.pkg -target /
+```
+
+### What Gets Installed
+
+The installer places files in:
+- `/usr/local/bin/ramalama` - Main executable
+- `/usr/local/share/ramalama/` - Configuration files
+- `/usr/local/share/man/` - Man pages
+- `/usr/local/share/bash-completion/` - Bash completions
+- `/usr/local/share/fish/` - Fish completions
+- `/usr/local/share/zsh/` - Zsh completions
+
+### Verify Installation
+
+```bash
+# Check version
+ramalama --version
+
+# Get help
+ramalama --help
+```
+
+## Method 2: Python Package (pip)
+
+If you prefer to use Python package management:
+
+```bash
+# Install Python 3.10 or later (if not already installed)
+brew install python@3.11
+
+# Install ramalama
+pip3 install ramalama
+
+# Or install from source
+git clone https://github.com/containers/ramalama.git
+cd ramalama
+pip3 install .
+```
+
+## Method 3: Build from Source
+
+For developers or if you want the latest code:
+
+```bash
+# Clone the repository
+git clone https://github.com/containers/ramalama.git
+cd ramalama
+
+# Install build dependencies
+pip3 install build
+
+# Build and install
+make install
+```
+
+## Prerequisites
+
+Before using RamaLama, you'll need a container engine:
+
+### Option A: Podman (Recommended)
+
+```bash
+brew install podman
+
+# Initialize Podman machine with libkrun for GPU access
+podman machine init --provider libkrun
+podman machine start
+```
+
+For more details, see [ramalama-macos(7)](ramalama-macos.7.md).
+
+### Option B: Docker
+
+```bash
+brew install docker
+```
+
+## Building the Installer Package (For Maintainers)
+
+If you want to build the installer package yourself:
+
+```bash
+# Install PyInstaller
+pip3 install pyinstaller
+
+# Build the package
+./scripts/build_macos_pkg.sh
+
+# The built package will be in:
+# build/macos-pkg/RamaLama-VERSION-macOS-Installer.pkg
+```
+
+## Uninstallation
+
+To remove RamaLama:
+
+```bash
+# Remove the executable
+sudo rm /usr/local/bin/ramalama
+
+# Remove configuration and data files (optional)
+sudo rm -rf /usr/local/share/ramalama
+rm -rf ~/.local/share/ramalama
+rm -rf ~/.config/ramalama
+
+# Remove man pages (optional)
+sudo rm /usr/local/share/man/man1/ramalama*.1
+sudo rm /usr/local/share/man/man5/ramalama*.5
+sudo rm /usr/local/share/man/man7/ramalama*.7
+
+# Remove shell completions (optional)
+sudo rm /usr/local/share/bash-completion/completions/ramalama
+sudo rm /usr/local/share/fish/vendor_completions.d/ramalama.fish
+sudo rm /usr/local/share/zsh/site-functions/_ramalama
+```
+
+## Troubleshooting
+
+### "ramalama: command not found"
+
+Make sure `/usr/local/bin` is in your PATH:
+
+```bash
+echo 'export PATH="/usr/local/bin:$PATH"' >> ~/.zshrc
+source ~/.zshrc
+```
+
+### "Cannot verify developer" warning
+
+macOS may show a security warning for unsigned packages. To bypass:
```

Comment:
I'd add a sentence on saying we're working on getting keys to sign it or something, we should not only ship unsigned pkgs.